src/sql/workbench/services/objectExplorer/browser/filterDialog/filterDialog.ts

```diff
@@ -464,10 +463,13 @@ export class ObjectExplorerServiceDialog extends Modal {
 					this._dialogService.show(Severity.Error, localize('filterDialog.errorEndDate', "End date is not specified."));
 					return;
 				}
-			}
-
-			if (filter.value !== '') {
-				this._treeNode.filters.push(filter);
+				if (value1 !== '' && value2 !== '') {
```

Comment:
I suppose having an empty value isn't a scenario you're concerned with right now? If we're focusing on just stuff like name or schema I guess that makes sense - but what about fields that may have empty values?

Reply:
https://github.com/microsoft/sqltoolsservice/blob/fcaca65a14b4570309be839d2aa933c117109c47/src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodesDefinition.xml#L584

Current that is something I am not trying to support